Modules/Test/classes/class.ilTestEvaluationGUI.php

```diff
@@ -315,7 +316,7 @@ public function outEvaluation()
                 try {
                     $globalCertificatePrerequisites = new ilCertificateActiveValidator();
                     if ($globalCertificatePrerequisites->validate()) {
-                        $options[] = $this->ui_factory->button()->shy($this->lng->txt('exp_grammar_as') . $this->lng->txt('exp_type_certificate'), $this->ctrl->getLinkTarget($this, 'certificate'));
+                        $options[] = $this->ui_factory->button()->shy($this->lng->txt('exp_grammar_as') . ' ' . $this->lng->txt('exp_type_certificate'), $this->ctrl->getLinkTarget($this, 'exportCertificateArchive'));
```

Comment:
@fhelfer There was a missing space for THIS option introduced with the latest changes.